homeassistant/components/fressnapf_tracker/quality_scale.yaml

```diff
@@ -26,22 +26,31 @@ rules:
   unique-config-entry: done
 
   # Silver
-  action-exceptions: todo
+  action-exceptions:
+    status: exempt
+    comment: |
+      No custom actions are defined.
```

Comment:
Turning on a light or switch also is an action

Reply:
Okay. I will handle this in a separate PR